docs/source/reference/frameworks/torchscript.rst

```diff
@@ -0,0 +1,23 @@
+===========
+TorchScript
+===========
+
+.. admonition:: About this page
+
+   This is an API reference for TorchScript in BentoML. As TorchScript
+   is a serialization format for PyTorch models, please refer to
+   :doc:`/frameworks/pytorch` for more information on how to use PyTorch
```

Comment:
Why link to pytorch and not torchscript?

Reply:
because torchscript is a part of pytorch, rather than a standalone library. Actually it should be libtorch instead of torchscript 🤔

Reply:
I think we should either reword this for a concrete reason to look at pytorch instead or remove it.

Reply:
updated. PTAL @sauyon